src/main/java/eggventory/model/states/FutureList.java

```diff
@@ -0,0 +1,67 @@
+package eggventory.model.states;
```

Comment:
they appear to be the same. hmm. 
couldnt history and future lists be implemented as instances of the same type of stateList? or perhaps both inherit the same superclass

Reply:
true, oh well, ill oopify it